packages/terra-tabs/src/StructuralTabs.scss

```diff
@@ -155,6 +156,7 @@
 
     .tab-menu:focus {
       background-color: var(--terra-tabs-structural-focus-background-color, #fff);
+      color: #000;
       color: var(--terra-tabs-structural-focus-color, initial);
```

Comment:
We might be able to use 'unset' for this.

Reply:
Unfortunately that isn't supported by Internet Explorer either

Reply:
BOOO!!! IE! BOOOOOOOOO!

Reply:
[](http://i.imgur.com/8m9mzPf.gif)

Reply:
![](https://i.gifer.com/7YAb.gif)

Reply:
Could you explain why you added the following? Is this what the original value is? I assume you grabbed this from terra-base?
```
color: #000;
```

Reply:
I believe it is the default color for most browsers.

Interestingly, it is used just as much in terra-core as the color set by terra-base, `#1c1f21`.